cvat/apps/dataset_manager/formats/mask.py

```diff
@@ -14,11 +18,36 @@
 from .registry import dm_env, exporter, importer
 from .utils import make_colormap
 
+class RotatedBoxesToPolygons(ItemTransform):
+    def _rotate_point(self, p, angle, cx, cy):
+        [x, y] = p
+        rx = cx + math.cos(angle) * (x - cx) - math.sin(angle) * (y - cy)
+        ry = cy + math.sin(angle) * (x - cx) + math.cos(angle) * (y - cy)
+        return rx, ry
+
+    def transform_item(self, item):
+        annotations = item.annotations
+        anns = [p for p in annotations if p.type == datum_annotation.AnnotationType.bbox and p.attributes['rotation']]
```

Comment:
```suggestion
        annotations = item.annotations[:]
        anns = [p for p in annotations if p.type == datum_annotation.AnnotationType.bbox and 'rotation' in p.attributes]
```

Reply:
'rotation' always in p.attributes, but it can be 0. In this case we should not convert boxes to polygons.